contracts/6/forwarder/ERC20ForwarderStorage.sol

```diff
@@ -15,7 +14,7 @@ import "./ERC20ForwardRequestCompatible.sol";
  *
  */
 
-contract ERC20ForwarderStorage is ERC20ForwardRequestTypes,Ownable {
```

Comment:
Where is the Ownable imported?

Reply:
I did make the ERC20Forwarder ownable. since it is importing BiconomyForwarder with custom Ownable it is expecting same in ERC20Forwarder as well. Shall discuss this